app/src/full/java/io/homeassistant/companion/android/vehicle/HaCarAppService.kt

```diff
@@ -65,14 +70,21 @@ class HaCarAppService : CarAppService() {
                 1
             )
 
+            @RequiresApi(Build.VERSION_CODES.P)
```

Comment:
Android Auto is available on versions 8+ (Oreo). As a result, this annotation should be removed and car info should do a version check instead of removing Auto support entirely for Android 8..